exe/bolt

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
+if RbConfig::CONFIG['host_os'] =~ /mswin|mingw|cygwin/
+  ENV['BOOTSNAP_CACHE_DIR'] = File.join(ENV['ProgramData'], 'PuppetLabs', 'bolt')
+  ENV['DISABLE_BOOTSNAP_LOAD_PATH_CACHE'] = true
+  require 'bootsnap/setup'
```

Comment:
```suggestion
  require 'bootsnap'
  Bootsnap.setup(
    cache_dir:       File.join(ENV['ProgramData'], 'PuppetLabs', 'bolt'),
    load_path_cache: false
  )
```

Bootsnap recommends manual setup if you're not using Rails (see [here](https://github.com/Shopify/bootsnap#usage)). Personally, I'd prefer to have more direct control over setup instead of relying on env vars, but am fine leaving this as-is if you have concerns.

Reply:
I actually had some trouble getting this working when it was packaged, let me test it first.

Reply:
Ok, confirmed this works - the original issue was that `setup` would only accept all five options, so was throwing `compile_cache_iseq not found` (or something along those lines). But this seems to work :+1: